src/integrationTest/java/uk/gov/hmcts/darts/task/service/AdminPatchAutomatedTaskTest.java

```diff
@@ -38,7 +38,7 @@ void allowsSuperAdminToPatchAutomatedTasks(SecurityRoleEnum role) throws Excepti
                 patch(ENDPOINT + "/1")
                     .contentType(APPLICATION_JSON_VALUE)
                     .content("""
-                                 { "is_active": false }
+                                 { "is_active": true }
```

Comment:
Author note: this test was changing a predefined automated task permanently, disabling it

Reply:
It would be better to keep the original intent of the test here, i.e we want to affect a change in state (changing `true` to `false`), whereas technically this test is having no effect on state (`true` to `true`).

Per the new guidance, I think this means we need to create a new automated task in a setup phase, and then patch it.

Reply:
I see, yes you're right. We can also more simply undo the changes to the predefined one. As long as that's not permanently deleted it's fine